src/app/(pages)/services/[serviceName]/page.tsx

```
@@ -0,0 +1,123 @@
+              }}
+            >
+              {currentEstablishment.address}
+            </Typography>
+          </Box>
+          <Box sx={{ position: 'relative' }}>
+            <iframe
+              src={`https://salvador-airport-preview.blumaps.com.br/?location=${currentEstablishment.locationId}`}
+              style={{ width: '100%', height: '42vh', borderRadius: '14px' }}
+            />
+
+            <Link
```

Biome (1.9.4)

[error] Provide a title attribute when using iframe elements.

Screen readers rely on the title set on an iframe to describe the content being displayed.

(lint/a11y/useIframeTitle)